### ISSUE TITLE
Library license question

### ISSUE THREAD

**Issue by machdyne** [mithro/kicad-pmod#4]
Hello and sorry for this potentially boring issue, but I'm trying to make sure that I am providing the correct licensing information for my projects and I am trying to understand the Apache 2.0 license in the context of this library.

I'm using parts from this library in a number of projects, distributed only as kicad_pcb and kicad_sch files. 

I'm wondering if it is necessary to distribute the Apache 2.0 license in my projects under those circumstances. I'm also wondering what sort of attribution, if any, is expected and necessary.

While I've got no problem with either, it does make things more complicated, especially for downstream projects, so I thought I would ask.

**Comment by omixann** [comment]
I'm also interested in knowing the answer to these questions, and I'd really like to know whether I can use this library on a project licensed under the CERN OHL terms (any of its variants).

**Comment by mithro** [comment]
Apache 2.0 is a permissive "do what you want" type license which is broadly compatible with most licenses out there. There is some information at https://choosealicense.com/licenses/apache-2.0/ - There is also info on Wikipedia at https://en.wikipedia.org/wiki/Apache_License#Licensing_conditions

If including this work in another work, you just need to make sure the license info is correctly kept with the files and if you have a NOTICE file include the info in that.

Hope that helps!

<blockquote><img src="/favicon.ico" width="48" align="right"><div>Choose a License</div><div><strong><a href="https://choosealicense.com/licenses/apache-2.0/">Apache License 2.0</a></strong></div><div>A permissive license whose main conditions require preservation of copyright and license notices. Contributors provide an express grant of patent rights. Licensed works, modifications, and larger works may be distributed under different terms and without source code.</div></blockquote>
<blockquote><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/88/The_ASF_Logo_%282025%29.svg/1280px-The_ASF_Logo_%282025%29.svg.png" width="48" align="right"><div><strong><a href="https://en.wikipedia.org/wiki/Apache_License#Licensing_conditions">Apache License - Wikipedia</a></strong></div></blockquote>

**Comment by omixann** [comment]
Makes sense, so I should be able to license my project under the terms I want as long as I retain your license information and clearly point to it. That's perfect!

I'm always a bit scared when talking about legal matters so I'm still glad I asked.
